### PR TITLE
#204 Improve user profile

### PR DESCRIPTION
**Pull request by Sontsevolosa**
Resolves [github issue](https://github.com/pivorakmeetup/pivorak-web-app/issues/204)

### Description

- Add count of sign in to the user profile
- Add 'pk-container' class to the user profile page
- Add localization for text
- Add styles from form for user sign in
- Add test for check if page have 'Count of sign in' text

### How to test instructions

rspec spec/features/profile

### Pre-merge checklist
 
- [ ] The PR relates to a single subject with a clear title and description
- [ ] Verify that feature branch is up-to-date with `development` (if not - rebase it). 

### Screenshots

profile_before
profile_after
edit_before
edit_after

### CODE REVIEW

File: app/views/members/_profile.slim [hunk 1]
```diff
@@ -1,3 +1,5 @@
 h2 = member.full_name
-
 = gravatar_image(member)
+h5
+  = t('members.count_of_sign_in')
+  = current_user.sign_in_count
```
Comment (denys-medynskyi) [inline app/views/members/_profile.slim]: Do we really need this sign in counter?

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]
Everything looks good!
However, I have 3 questions/suggesstions:
- [ ] Do we need sign in counter?
- [ ] How can I access this page via link?
- [ ] Please, rebase your branch on master to make CI pass.

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review my changes)

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]
Can you add the same color background as [here](https://pivorak.com/users/sign_in)  and put fields on the center of the screen?
By the way, you can argue with me if you disagree.

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review my code)

File: .travis.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-language:
```
Comment (denys-medynskyi) [inline .travis.yml]: Just rebase this branch on development, so you will not need to add this manually.

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review my code

File: app/views/profile/edit.slim [hunk 3]
```diff
@@ -1,23 +1,30 @@
-.pk-profile-edit
-  - title t('members.edit_profile')
+.pk-auth
+  h2.pk-uppercase = t('members.edit_profile')
+  = simple_form_for [current_user], url: profile_path, html: { class: 'pk-form' } do |f|
+    /<div style="
```
Comment (denys-medynskyi) [inline app/views/profile/edit.slim]: Remove commented code

**Review by denys-medynskyi — CHANGES_REQUESTED** [review]
After update profile notification message is wrong.
Can you fix it?
After update I see this - https://cl.ly/3y2V1L2W0J3g.
I think it just needs some string in en.yml file

**Comment by denys-medynskyi** [comment]
Another thing - pivorak logo is missing on edit profile form - https://cl.ly/1k0422412z1u

**Comment by denys-medynskyi** [comment]
This page looks ugly - https://cl.ly/0L2V2w1M3z0Z, so make `profile` link lead to `/profile/edit` or improve profile page

**Comment by sveredyuk** [comment]
@denysmedynskyi Please..... man..... bro..... please.... review!

**Comment by Sontsevolosa** [comment]
@denysmedynskyi please, review changes)

**Comment by denys-medynskyi** [comment]
@sveredyuk are you asking me to review other PRs?

**Review by denys-medynskyi — APPROVED** [review]

**Pull request merged**